### PR TITLE
Adding GitHub

### PR DESCRIPTION
**Pull request by kinlane**
Adding the GitHub RESTFest organization members as JSON in _data

### CODE REVIEW

**Comment by BigBlueHat** [comment]
Can we move these into `_data/raw/github` (and maybe another sub-folder to explain what they are)?

I'm finding my linked data brain is very sad at how none of that JSON actually says what it is or how the items in the result array are related. 😕 

I suppose what I'm mentally wanting (and in practice one way or the other) is this resource: https://api.github.com/orgs/restfest with `members_url`'s contents (already in this PR) added into a `members` property...or something.

**tl;dr** I'm totally fine to ship this, but will continue to day dream about better data representations all the places. 😜
